cli/Cargo.toml

```diff
@@ -27,7 +27,7 @@ sgx_crypto_helper = { branch = "master", git = "https://github.com/apache/teacla
 substrate-api-client = { features = ["ws-client"], git = "https://github.com/scs/substrate-api-client", branch = "polkadot-v0.9.19" }
 substrate-client-keystore = { git = "https://github.com/scs/substrate-api-client", branch = "polkadot-v0.9.19" }
 my-node-runtime = { git = "https://github.com/integritee-network/integritee-node", branch = "master", package = "integritee-node-runtime" }
-teerex-primitives = { git = "https://github.com/integritee-network/pallets.git", branch = "master" }
+teerex-primitives = { git = "https://github.com/integritee-network/pallets.git", branch = "gk-block-finalization" }
```

Comment:
The idea here, is that we use this branch until your changes in the pallets repo are merged into master?

Reply:
yes, I wanted to prepare that pallet change can run throuh the CI before merging